src/rusty_hook.rs

```diff
@@ -62,7 +61,7 @@ pub fn run<F, G, H, I>(
     read_file: H,
     log: I,
     hook_name: &str,
-    _args: &str,
+    args: &str,
```

Comment:
Bit of a nit, but can we change this to just take the parsed Option with the owned string and tweak `main.rs` accordingly?

Most hooks don't have any git params and I don't think we should be doing the token substitution in cases where there are no git args (e.g. if someone has `pre-commit = "echo %@"` then we shouldn't be replacing `%@` with an empty string/removing it from the hook script).

This obviously isn't performance critical code, but by using the Option we can avoid having to run replace and the associated copy when there's really nothing for us to replace with anyway. Will also make it easier to support the desired environment variable approach too (refs #25)

What do you think?

Reply:
Done!